pkg/monitors/appinsights/ingress-test.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
```

Comment:
Lets move these configs examples into a readme in separate file under docs/ and remove this example file. and add its refernce under main README so it becomes easier to find config paramaters